src/components/learner-credit-management/data/utils.js

```diff
@@ -564,6 +565,13 @@ export const isLmsBudget = (
  */
 export const hasCourseStarted = (start) => dayjs(start).isBefore(dayjs());
 
+export function isArchived(courseRun) {
```

Comment:
nit: (discussed already, but posting for posterity) no longer used with `isLateEnrollmentEligible`